DIVERGENCES.md

```diff
@@ -18,6 +18,13 @@ For interfaces to be considered analogous, there should be some evidence that de
 - **Node.js API**: [Node Streams](https://nodejs.org/api/stream.html)
 - **Mitigation strategies**: [whatwg-stream](https://github.com/nodejs/whatwg-stream) effort to implement WHATWG Streams as a Node.js module; [`readable-stream`](https://www.npmjs.com/package/readable-stream) provides Node streams for the Web
 
+## Buffers
+
+- **Purpose**: Interaction with octet streams
+- **Web API**: [ArrayBuffer API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
```

Comment:
I believe equivalent of Node.js `Buffer` is `Uint8Array`, or `DataView`, or at least `ArrayBufferView`?